src/test/rustdoc/intra-doc-crate/auxiliary/hidden.rs

```diff
@@ -0,0 +1,19 @@
+#![crate_name = "hidden_dep"]
+#![deny(intra_doc_resolution_failure)]
```

Comment:
I made the same mistake in my PR. We should probably have `deny(unknown_lints)` on by default in UI-tests.

```suggestion
#![deny(intra_doc_link_resolution_failure)]
```